### PR TITLE
Modernize code

### PR DESCRIPTION
**Pull request by rubensworks**
This PR does two things:

* Migrate CI to GH Actions (as Travis does not work anymore)
* Migrate RDF/JS typings to their new location

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-test-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
+        run: yarn install
+      - name: Build project
+        run: yarn run build
+      - name: Run tests
+        run: yarn run test
```
Comment (joachimvh) [inline .github/workflows/ci.yml:32]: Would want to use npm here since this package uses a `package-lock.json`

**Review by joachimvh — CHANGES_REQUESTED** [review]
Thanks for doing this, only change is that I would want to use npm to be consistent with the lockfile.

Also let me know if a release is needed after this for Comunica or not.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rubensworks) [reply .github/workflows/ci.yml:32]: Oh right. I just copy-pasted this from my other projects 😅

**Comment by rubensworks** [comment]
Changes pushed.
A new release would definitely be appreciated! :-) (I'm just doing a minor version bump for these typing changes)

**Review by joachimvh — APPROVED** [review]

**Pull request merged**